plugins/upload-assets-to-s3/README.md

```diff
@@ -0,0 +1,79 @@
+# @dotcom-tool-kit/upload-assets-to-s3
+
+Upload files to a configured AWS S3 bucket.
+
+## Installation
+
+Install `@dotcom-tool-kit/upload-assets-to-s3` as a `devDependency` in your app:
+
+```sh
+npm install --save-dev @dotcom-tool-kit/upload-assets-to-s3
+```
+
+Add the plugin to your [Tool Kit configuration](https://github.com/financial-times/dotcom-tool-kit/blob/main/readme.md#configuration):
+
+```yaml
+plugins:
+	- '@dotcom-tool-kit/upload-assets-to-s3'
+```
+
+Install this plugin's tasks:
+
+```sh
+npx dotcom-tool-kit --install
+```
+
```

Comment:
**issue**: this plugin doesn't have add any hooks so we don't need to install anything new

```suggestion
```